pkg/provider/resource_cluster.go

```diff
@@ -35,47 +35,47 @@ func (c *ClusterResource) Schema() *schema.Resource {
 
 		Schema: map[string]*schema.Schema{
 			"allowed_ip_ranges": {
-				Description: "Allowed IP ranges",
+				Description: "Allowed IP ranges.",
 				Type:        schema.TypeList,
 				Optional:    true,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"cidr_block": {
-							Description: "CIDR Block",
+							Description: "CIDR Block.",
 							Type:        schema.TypeString,
 							Required:    true,
 						},
 						"description": {
-							Description: "CIDR Block Description",
+							Description: "CIDR Block Description.",
 							Type:        schema.TypeString,
 							Optional:    true,
 						},
 					},
 				},
 			},
 			"backup_retention_period": {
-				Description: "Backup Retention Period.",
+				Description: "Backup Retention Period. e.g. \"7d\", \"2w\" or \"3m\".",
 				Type:        schema.TypeString,
 				Optional:    true,
 			},
 			"cluster_architecture": {
-				Description: "Cluster Architecture",
+				Description: "Cluster Architecture. See https://www.enterprisedb.com/docs/biganimal/latest/overview/02_high_availability/ for details.",
```

Comment:
```suggestion
				Description: "Cluster Architecture. See [Supported cluster types ](https://www.enterprisedb.com/docs/biganimal/latest/overview/02_high_availability/) for details.",
```

Reply:
Clickable link is added at https://github.com/EnterpriseDB/terraform-provider-biganimal/commit/0aea7392df32d1d3b7fcb6e35524aa585fa8a660